### PR TITLE
Fix express checkout button order

### PR DESCRIPTION
**Pull request by brettshumaker**
#### Changes proposed in this Pull Request

#5520 introduced changes to the separator displayed between express checkout buttons and the CTA button (or checkout form depending on the page) but the order of the buttons changed. This PR fixes the order so that the WooPay button is listed first.

| Before | After |
|-----|-----|
| after | after |

#### Testing instructions

* Apply patch
* Enable WooPay as well as Apple Pay and Google Pay.
* Visit any page that shows the express payment buttons (single product, cart, checkout) and ensure that WooPay is listed first. 
-------------------

- [x] Run `npm run changelog` to add a changelog file, choose `patch` to leave it empty if the change is not significant. You can add multiple changelog files in one PR by running this command a few times. 
- [ ] Covered with tests (or have a good reason not to test in description ☝️)
- [x] Tested on mobile (or does not apply)

**Post merge**

- [ ] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) following [these instructions](https://github.com/Automattic/woocommerce-payments/wiki/How-to-write-good-manual-testing-scenarios) : _Add link here / 'QA Testing Not Applicable'_
- [ ] Add or update [critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Critical-flows) and [testing instructions for critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Testing-instructions-for-critical-flows), if applicable.
- [ ] Add what's changed (description, screenshot, demo videos etc.) to the release announcement post, if applicable.

### CODE REVIEW

**Comment by asumaran** [comment]
@brettshumaker, what about the buttons in the checkout block? It hasn't changed compared to the previous WCPay version, but I noticed the WooPay button appears after GPay.

Screenshot 2023-02-17 at 2 36 24 PM

**Review by asumaran — APPROVED** [review]
Looks good! Tests as expected. I left a comment that probably belongs to the WC Blocks extension.

**Comment by brettshumaker** [comment]
> what about the buttons in the checkout block? It hasn't changed compared to the previous WCPay version, but I noticed the WooPay button appears after GPay.

@asumaran  Good catch! I looked into this a bit and it looks like if I change the order in which we register the express payment buttons, I can get WooPay to show up first. Essentially moving this [block of code](https://github.com/Automattic/woocommerce-payments/blob/e04ea0dd455d48123dd3de87d83d520b0deed6ee/client/checkout/blocks/index.js#L70-L81) above line 68 in that file. I'm going to see if there's a better way to do this, but that did work for me.

File: client/checkout/blocks/index.js [hunk 1]
```diff
@@ -65,8 +65,6 @@ registerPaymentMethod( {
 	},
 } );
 
-registerExpressPaymentMethod( paymentRequestPaymentMethod( api ) );
-
 // Call handlePlatformCheckoutEmailInput if platform checkout is enabled and this is the checkout page.
 if ( getConfig( 'isPlatformCheckoutEnabled' ) ) {
```
Comment (brettshumaker) [inline client/checkout/blocks/index.js:69]: @asumaran I didn't see a way to explicitly set the ordering of these methods with `registerExpressPaymentMethod()` so I went ahead with changing the order in which we register them. LMK your thoughts on this approach.

Comment (asumaran) [reply client/checkout/blocks/index.js:69]: I see no harm on doing this.

**Review by asumaran — APPROVED** [review]
LGTM!

**Pull request merged**